src/main/java/tp/cap5buddy/contacts/ContactList.java

```diff
@@ -0,0 +1,91 @@
+package tp.cap5buddy.contacts;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import tp.cap5buddy.contacts.exceptions.ContactNotFoundException;
+import tp.cap5buddy.contacts.exceptions.DuplicateContactException;
+
+public class ContactList {
+
+    public final List<Contact> contacts;
```

Comment:
Should it be final? Might be hard to keep updating right?

Reply:
Even if an object of List type is declared final, we can still add items into the list. We are only unable to overwrite the list with an entirely new list.